icons/draugiem-dot-lv.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Draugiem.lv icon</title><path d="M8.169 15.089c.048.346.14.562.278.692.143.13.34.18.595.21.304-.03.51-.1.65-.3.083-.13.14-.33.166-.61l.098-1.159v-.02l-.02-.18-.03-.17-.03-.175-.032-.165-.03-.174-.03-.165-.03-.164-.03-.17-.028-.16-.03-.15-.03-.15v-.137l-.03-.136-.03-.133-.015-.126-.03-.116-.03-.1-.03-.17-.047-.167-.045-.18-.045-.18-.013-.197-.052-.203-.045-.21-.053-.21-.03-.21-.043-.215-.03-.213-.017-.242-.015-.21-.01-.227v-.402l.02-.21.035-.21.046-.21.06-.2.09-.194.106-.186.04-.074c-.2-.046-.44-.076-.69-.076-.11 0-.2 0-.3.016-2.329.2-1.38 2.835-.98 4.282.28 1 .6 3.339.72 4.059l-.02-.02zm.28-8.662c.09.03.177.045.282.045.48 0 .79-.255.96-.645a2.35 2.35 0 00.153-.944 2.634 2.634 0 00-.164-.81c-.195-.467-.54-.782-1.066-.782-.073 0-.134 0-.18.016-.674.11-.975.78-.944 1.61.015.78.314 1.43.93 1.58l.028-.06zm1.243 3.842c.084.36.168.69.237.975.285 1.155.562 3.842.665 4.666.098.826.39.99.97 1.05.575-.044.866-.224.964-1.034.097-.81.38-3.497.657-4.651.068-.28.16-.61.24-.96.258-1.05.466-2.29 0-3.099-.296-.51-.865-.84-1.86-.84-1.006 0-1.582.33-1.87.86-.466.81-.257 2.06 0 3.109l-.003-.06zM11.55 5.99h.015c.9 0 1.33-.822 1.33-1.83 0-1.006-.42-1.845-1.32-1.845h-.01c-.91 0-1.33.84-1.33 1.86s.42 1.84 1.337 1.84v-.03zm2.826.752c-.27 0-.51.03-.705.075l.045.075.105.194.075.2.06.21.045.21.03.21.015.23v.44l-.015.23v.166L14 9.2l-.03.213-.044.213-.046.21-.044.21-.045.206-.05.2-.05.197-.06.195-.05.18-.03.18-.05.168-.018.09-.03.12-.03.137-.026.134-.015.132-.03.12v.03l-.03.15-.03.15-.02.162-.03.168-.02.165-.03.165-.02.18-.013.165-.014.18-.014.18-.017.18v.014c.045.496.075.93.09 1.17.03.287.075.48.166.615.14.2.35.27.66.3.26-.03.45-.07.6-.22.15-.12.23-.342.29-.69.12-.702.44-3.048.71-4.051.4-1.45 1.35-4.082-.98-4.283-.105-.014-.195-.014-.3-.014l.012-.05zm.016-.27c.09 0 .18-.014.27-.03.6-.15.9-.794.93-1.56.03-.81-.27-1.499-.95-1.59-.06-.017-.12-.017-.18-.017-.54 0-.89.33-1.07.79-.09.24-.15.52-.15.82-.02.36.03.69.15.96.16.4.48.66.96.66l.03-.05zm2.789 1.2c4.289.062 6.798 2.701 3.959 6.303-1.32 1.667-3.53 3.316-6.488 4.367l-.42.15c-.29.09-.56.18-.83.256-.65.18-1.26.344-1.86.464-.65.14-1.279.23-1.879.3-.44.05-.86.08-1.25.08-2.746.08-4.816-.67-5.488-2.279-.81-1.92.823-4.589 4.109-6.628-2.43.97-6.838 3.289-7.034 6.568v.42c.016.35.074.68.195 1.04.707 2.219 4.009 3.269 8.208 2.899.406-.03.826-.073 1.25-.14.604-.082 1.229-.21 1.859-.36.604-.13 1.23-.31 1.864-.51l.42-.13c.285-.09.557-.18.826-.285 3.645-1.32 6.348-3.074 7.923-5.1.9-1.152 1.365-2.232 1.455-3.18v-.732c-.26-2.412-3.06-3.823-6.875-3.434l.044-.045z"/></svg>
```

Comment:
There seem to be some differences between PR (red) and source (black).
Any chance you could look at a rebuild? I've provided my built-from-source 3 decimal path below:
```svg
M17.128 7.695c4.301.054 6.808 2.692 3.961 6.303-1.307 1.66-3.518 3.302-6.468 4.362l-.425.144a23.554 23.554 0 01-2.687.715c-.643.131-1.266.226-1.869.296-.43.041-.847.069-1.248.077-2.741.069-4.805-.679-5.477-2.273-.811-1.917.831-4.592 4.118-6.627C4.599 11.659.185 13.975 0 17.259v.417c.022.35.087.695.193 1.029.7 2.219 4.002 3.248 8.202 2.883.409-.036.824-.082 1.248-.144a24.088 24.088 0 003.732-.875c.139-.049.278-.09.417-.136.283-.09.561-.188.831-.291 3.642-1.323 6.344-3.088 7.922-5.106.901-1.156 1.367-2.237 1.454-3.186v-.726c-.249-2.404-3.054-3.817-6.871-3.429zm-2.792-1.184c.096-.001.192-.013.286-.036.607-.152.906-.803.934-1.573.028-.816-.257-1.495-.934-1.593a.774.774 0 00-.18 0 1.114 1.114 0 00-1.068.79 2.552 2.552 0 00-.152.816c-.021.326.031.653.152.957a.998.998 0 00.962.639zm-.021.258a2.944 2.944 0 00-.7.075l.057.077.103.188.082.193.069.201.049.208.036.206.021.216v.651L14.019 9l-.028.214-.028.214-.041.216-.042.213-.044.208-.046.208-.041.201-.049.194-.049.188-.049.18-.041.172-.041.167-.021.103-.028.118-.028.124-.021.131-.028.139-.021.118v.028l-.028.152-.026.152-.021.16-.028.167-.021.165-.028.168-.021.172-.028.167-.021.172-.021.172-.021.172c.041.499.077.921.098 1.163.002.214.059.424.165.61.155.2.398.314.651.304a.968.968 0 00.597-.216c.164-.195.261-.438.275-.692.111-.713.43-3.055.708-4.059.394-1.446 1.343-4.079-.983-4.28a2.574 2.574 0 00-.305-.016zm-2.831-.741c.901 0 1.323-.831 1.323-1.843s-.396-1.838-1.297-1.845c-.927 0-1.349.826-1.349 1.856s.422 1.838 1.336 1.838l-.013-.006zM9.628 10.3c.082.353.165.679.234.97.286 1.156.561 3.845.667 4.653.106.808.386.991.968 1.048.577-.057.867-.229.963-1.048.095-.818.394-3.493.672-4.648.069-.291.16-.618.242-.97.257-1.06.463-2.316 0-3.109-.299-.515-.865-.852-1.863-.852s-1.578.34-1.869.852c-.474.803-.257 2.049 0 3.109l-.014-.005zM8.395 6.468c.093.023.188.035.283.036.428.011.815-.251.965-.651.115-.306.167-.632.153-.958a2.803 2.803 0 00-.152-.816 1.123 1.123 0 00-1.068-.79.774.774 0 00-.18 0c-.672.098-.963.772-.935 1.593.018.783.325 1.434.934 1.586zm-.275 8.658c.01.255.107.5.275.692a.968.968 0 00.597.216.798.798 0 00.651-.299c.099-.189.155-.397.165-.61l.098-1.156v-.015l-.021-.172-.02-.172-.028-.172-.021-.168-.021-.172-.028-.168-.021-.165-.028-.167-.036-.167-.021-.152-.026-.152v-.021l-.021-.126-.028-.139-.027-.131-.021-.124-.028-.118-.028-.103-.033-.167-.049-.172-.041-.18-.049-.188-.05-.193-.049-.201-.044-.201-.048-.208-.041-.214-.034-.216-.028-.214-.029-.213-.02-.217v-.643l.021-.216.036-.206.049-.208.062-.201.09-.193.103-.188.057-.077a3.057 3.057 0 00-.7-.075c-.1-.002-.199.002-.299.013-2.316.201-1.377 2.831-.983 4.28.286 1.004.605 3.346.715 4.059h.002z
```

![image](https://user-images.githubusercontent.com/1830867/102891339-eaf89280-4455-11eb-8ef7-044730321aa2.png)

Reply:
I've rebuilt it (forgot to mention than in my commit message - don't get confused) and then optimized it with a precision of 3. 👍🏼